script/build.ts

```diff
@@ -131,21 +130,19 @@ function packageApp() {
     )
   }
 
-  const toPackageArch = (targetArch: string | undefined): OfficialArch => {
-    if (targetArch === undefined) {
-      targetArch = os.arch()
+  const getPackageArch = (): OfficialArch => {
+    const arch = process.env.npm_config_arch || process.arch
+
+    if (arch === 'arm64' || arch === 'x64' || arch === 'armv7l') {
```

Comment:
I'd rather have `armv7l` here (and the other places) throw an error. like I said process.arch won't ever return it. thats not part of the source code. and npm_config_arch won't compile for arm if it is set to armv7l.
so it would be better for arch === `armv7l` to hit the error like it will if it is removed from this if statement.
```
    throw new Error(
      `Building Desktop for architecture '${arch}' is not supported`
    )
```

same goes for here